### PR TITLE
Add delegate-accepting UsingNamer overload

### PR DESCRIPTION
**Pull request by masaeedu**
Following on from #10.

### CODE REVIEW

**Comment by masaeedu** [comment]
There seems to be something fishy going on with the CI. I'm seeing:

> Building Assent v1.0.2-pullrequest0011-0003
> Build with specified version already exists.

File: src/Assent/Namers/DelegateNamer.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Assent.Namers
+{
+    class DelegateNamer : INamer
```
Comment (droyad) [inline src/Assent/Namers/DelegateNamer.cs]: This should be public

**Comment by masaeedu** [comment]
@droyad K, done.

**Comment by droyad** [comment]
Thanks @masaeedu , it has 🚢 ed

**Pull request merged**